### PR TITLE
Added languagetool server

### PR DESCRIPTION
**Pull request by 1Euro7Cent**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [ ] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:
  I don't know how

### New Server Submissions:

1. [x] Does your submission pass tests (server is connectable)?
2. [ ] Does your server use a custom docker image?
    * [ ] Have you tried to use a generic image?
    * [ ] Did you PR the necessary changes to make it work?
3. [x] Have you added the server to the main README.md?
4. [x] Have you added a unique README.md for the server you are adding?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -250,3 +250,5 @@ If you are reading this it looks like you are looking to add an egg to your serv
 * [haste-server](/software/haste-server)
 ### RabbitMQ
 * [rabbitmq](/software/rabbitmq)
+### Language-Tool
```
Comment (Software-Noob) [inline README.md:253]: The list is in alphabetical order. Put this above RabbitMQ. 

The name is LanguageTool, as seen on their website, not Language-Tool. Please correct this to match in both readmes you've created.

**Review by Software-Noob — CHANGES_REQUESTED** [review]

**Comment by Software-Noob** [comment]
I am not sure if you've tested your new change, but it breaks the egg since it tries to unarchive files without `ngrams` in the filename. The previous version worked fine with the suggestions I provided.

**Comment by 1Euro7Cent** [comment]
I tested it. On my site it works. And I dont download with ngrams at the file

**Comment by Software-Noob** [comment]
The download fails, because the host has Cloudflare enabled which blocks the download, and as such only, the Cloudflare HTML file is fetched. You might not have this if you're self-hosting, but most datacenter IPs are flagged causing downloads to fail (403 access denied).

I do see they have a Github for source code, but besides that, this egg will not work if everyone is blocked by the host.

```  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 12409    0 12409    0     0   299k      0 --:--:-- --:--:-- --:--:--  302k
unziping
Archive:  download.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of download.zip or
        download.zip.zip, and cannot find download.zip.ZIP, period.
moving files
cp: can't stat 'LanguageTool-*/*': No such file or directory
```

**Pull request closed without merging**